metadata/src/main/java/org/apache/kafka/controller/metrics/QuorumControllerMetrics.java

```diff
@@ -109,6 +114,15 @@ public Long value() {
                 return time.milliseconds() - lastAppliedRecordTimestamp();
             }
         }));
+
+        if (dualWriteModeState.get()) {
```

Comment:
This isn't quite right. As written, this atomic boolean won't get updated until we start dual-writing which comes much later than this class being constructed. We'll need to pass in a boolean from ControllerServer indicating if zk migrations are enabled.